src/style.css

```diff
@@ -0,0 +1,81 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+body {
+  margin: 0;
+}
+
+.conteiner {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.box {
+  height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-basis: 33.33%;
+}
+
+.box1 {
+  background-color: #f00;
+  order: 1;
+}
+
+.box2 {
+  background-color: #c00;
+  order: 2;
+}
+
+.box3 {
+  background-color: #900;
+  order: 3;
+}
+
+.box4 {
+  background-color: #600;
+  order: 6;
+}
+
+.box5 {
+  background-color: #300;
+  order: 5;
+}
+
+.box6 {
+  background-color: #000;
+  order: 3;
+}
+
+@media (max-width: 750px) {

```

Comment:
the width of your blocks must be at least 300 pixels, if you use a media query similar to yours, this does not satisfy the condition

Reply:
![image](https://user-images.githubusercontent.com/35385361/112841259-f25cb700-90a8-11eb-8265-bb2d2e3f38d7.png)